src/main/java/classify/student/StudentSorter.java

```diff
@@ -102,8 +136,20 @@ private static void listStudents(ArrayList<Student> students) {
      * @param students The list of students to be listed.
      */
     private static void listStudentsWithTotalClasses(ArrayList<Student> students) {
+        int i = 1;
+        for (Student student : students) {
```

Comment:
Perhaps you could remove line 139 and 143 by using the normal for loop, and make a new function in Ui class to print out the needed details.